### PR TITLE
install captcha bundle

### PR DESCRIPTION
**Pull request by itkg-rboessel**

### CODE REVIEW

**Comment by itkg-rboessel** [comment]
https://github.com/open-orchestra/open-orchestra-front-demo/pull/7

File: DisplayBundle/Resources/config/routing.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+captcha_routing:
+    resource: "@InnoceadCaptchaBundle/Resources/config/routing.yml"
```
Comment (nicolasThal) [inline DisplayBundle/Resources/config/routing.yml]: eof

File: composer.json [hunk 2]
```diff
@@ -12,7 +12,8 @@
     "require": {
         "symfony/swiftmailer-bundle" : "~2.3",
         "open-orchestra/open-orchestra-model-interface": "*",
-        "friendsofsymfony/http-cache-bundle": "1.2.1"
+        "friendsofsymfony/http-cache-bundle": "1.2.1",
+        "innocead/captcha-bundle": "*"
```
Comment (nicolasThal) [inline composer.json]: why did you choose this bundle?

**Comment by itkg-ngilain** [comment]
:+1:

**Pull request merged**